src/Validation/Validator.php

```diff
@@ -11,7 +11,7 @@
 
 final readonly class Validator
 {
-    public function validate(IsValidated $object): void
+    public function validate(object $object): void
```

Comment:
Will need to get rid of `IsValidated` entirely. Doesn't make sense for objects to implement an empty interface, just to be able to get validated.